tests/articulation_tests.ts

```diff
@@ -250,53 +250,59 @@ function verticalPlacement2(options: TestOptions, contextBuilder: ContextBuilder
 
   const notes = [
     staveNote({ keys: ['f/4'], duration: 'q' })
-      .addModifier(new Articulation('fermataBelow'), 0)
-      .addModifier(new Articulation('augmentationDot').setPosition(ModifierPosition.BELOW), 0)
-      .addModifier(new Articulation('articTenutoBelow'), 0),
+      .addModifier(new Articulation('\ue4c1' /*fermataBelow*/), 0)
+      .addModifier(new Articulation('\ue1e7' /*augmentationDot*/).setPosition(ModifierPosition.BELOW), 0)
+      .addModifier(new Articulation('\ue4a5' /*articTenutoBelow*/), 0),
```

Comment:
I think it is OK that we are using the unicode codepoints directly, for now.

However, developers would have to look up the code points in the SMuFL spec each time. It would be nice if there was an easy (readable) way to specify the glyphs we want. Maybe we can figure out the const enum thing at some point.

Reply:
Agree.